docs/deploy-custom-image.rst

```diff
@@ -566,3 +566,77 @@ If you are using a private repository to push your images to, make sure to pass
 Or pass it to :ref:`upload <commands-upload>` command::
 
     $ shub image upload --username johndoe --password yourpass
+
+
+Container works locally but fails in scrapy cloud
+-------------------------------------------------
+
+Prior to running ``start-crawl`` in Scrapy Cloud, some configurations
+are set to ensure we can run an isolated process.
+This can lead to issues that are quite hard to debug and find the
+root cause.
+To aid in this process, below you willl find some steps that
+are quite similar to what actually runs in scrapy cloud.
```

Comment:
```suggestion
Prior to running ``start-crawl`` in Scrapy Cloud, some configurations
are set to ensure we can run an isolated process.
This can lead to issues that are quite hard to debug and find the
root cause.
To aid in this process, below you will find some steps that
are quite similar to what actually runs in Scrapy Cloud.
```